src/docs/arch-metadata.dox

```diff
@@ -1,19 +1,78 @@
 /*! @arch_page arch-metadata Metadata
 
-Metadata in WiredTiger is stored as a table in the \c "WiredTiger.wt" file.
-The table's key is a \c uri string, and the value is its complete
-configuration string.  The configuration itself is a list of key/value
-pairs in string form.  The configuration's keys are dependent on the
-type of \c uri.  Thus, a metadata entry with a \c uri key beginning with
-\c "table:" will be a configuration string
-having configuration entries like \c key_format and \c value_format to
-describe the data encoding for the uri.  A metadata key beginning with
+Metadata in WiredTiger captures important information about the user's database.
+Metadata includes tracking essential information such as the files and tables
+present in the database, their associated configurations, as well as the latest checkpoints of
+each those files. In particular, the checkpoint metadata also tells WiredTiger where to the find
+root page and all other parts of its tree when accessing the file.
+
+The main metadata in WiredTiger is stored in the \c WiredTiger.wt file.
+
+@section metadata_table Metadata Table
+
+The keys in the \c WiredTiger.wt table are \c uri strings. The keys representing a
+range of the available URIs in the system. This for example can include \c "table:" & \c "file:"
+prefixed URIs, representing typical row-store WiredTiger tables. A non-exhaustive list of other
+example keys could also include \c "index:", \c "colgroups:" \& \c "system:" prefixed URIs, representing
+indexes, colgroups and special system values respectively.
+
+The value corresponding to a given \c uri key in the metadata table is its
+associated configuration string.  The configuration value itself is a list of key/value
+pairs in string form.  The key/value pairs in the configuration strings are dependent on the
```

Comment:
There is a `config-strings.dox` file that you should reference here.

Reply:
Great idea. I've updated the description with a reference to config strings doc: 170983e67b58c4d1194b991c5a58411df1029157